src/Microsoft.PowerShell.Commands.Utility/commands/utility/ObjectCommandComparer.cs

```diff
@@ -139,10 +139,12 @@ public override int GetHashCode()
     internal class ObjectCommandComparer : IComparer
     {
         /// <summary>
+        /// Initializes a new instance of the <see cref="ObjectCommandComparer"/> class.
         /// Constructor that doesn't set any private field.
```

Comment:
@iSazonov This text needs clarifying, the constructor is setting fields.

Reply:
@iSazonov Should we update the comment in this PR?

Reply:
I think the comment is correct but tricky - it means `((ICompare)obj1).CompareTo(obj2)` should works.

Reply:
OK, I have removed TODO.